Makefile

```diff
@@ -64,91 +101,183 @@ clean-test:
 	python setup.py bdist_wheel upload
 
+.PHONY: bump
+bump:
+	$(shell bash -c 'read -p "Version: " VERSION_PART; \
+		source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; \
+		test -f "$(CONDA_ENV_PATH)/bin/bumpversion" || pip install bumpversion; \
+		"$(CONDA_ENV_PATH)/bin/bumpversion" --config-file "$(CUR_DIR)/.bumpversion.cfg" \
+		--verbose --allow-dirty --no-tag --new-version $$VERSION_PART patch;')
+
+.PHONY: bump-dry
+bump-dry:
+	$(shell bash -c 'read -p "Version: " VERSION_PART; \
+		source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; \
+		test -f "$(CONDA_ENV_PATH)/bin/bumpversion" || pip install bumpversion; \
+		"$(CONDA_ENV_PATH)/bin/bumpversion" --config-file "$(CUR_DIR)/.bumpversion.cfg" \
+		--verbose --allow-dirty --dry-run --tag --tag-name "{new_version}" --new-version $$VERSION_PART patch;')
+
+.PHONY: bump-tag
+bump-tag:
+	$(shell bash -c 'read -p "Version: " VERSION_PART; \
+		source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; \
+		test -f $(CONDA_ENV_PATH)/bin/bumpversion || pip install bumpversion; \
+		"$(CONDA_ENV_PATH)/bin/bumpversion" --config-file "$(CUR_DIR)/.bumpversion.cfg" \
+		--verbose --allow-dirty --tag --tag-name "{new_version}" --new-version $$VERSION_PART patch;')
+
 .PHONY: dist
-dist: clean
+dist: clean conda-env
 	@echo "Creating distribution..."
-	python setup.py sdist
-	python setup.py bdist_wheel
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; python setup.py sdist'
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; python setup.py bdist_wheel'
 	ls -l dist
 
 .PHONY: sysinstall
-sysinstall: clean
+sysinstall: clean conda-env
 	@echo "Installing system dependencies..."
-	pip install --upgrade pip setuptools
-	pip install gunicorn
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; pip install --upgrade pip setuptools'
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; pip install gunicorn'
 
 .PHONY: install
 install: sysinstall
 	@echo "Installing Magpie..."
-	python setup.py install
+	# TODO: remove when merged
+	# --- ensure fix is applied
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; \
+		pip install --force-reinstall "https://github.com/fmigneault/authomatic/archive/httplib-port.zip#egg=Authomatic"'
+	# ---
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; pip install --upgrade "$(CUR_DIR)"'
+
+.PHONY: install-dev
+install-dev: conda-env
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; pip install -r "$(CUR_DIR)/requirements-dev.txt"'
+	@echo "Successfully installed dev requirements."
 
 .PHONY: cron
 cron:
 	@echo "Starting Cron service..."
 	cron
 
 .PHONY: start
-start: cron install
+start: install
 	@echo "Starting Magpie..."
-	exec gunicorn -b 0.0.0.0:2001 --paste "$(CUR_DIR)/magpie/magpie.ini" --workers 10 --preload
+	@bash -c 'source "$(CONDA_HOME)/bin/activate" "$(CONDA_ENV)"; \
+		exec gunicorn -b 0.0.0.0:2001 --paste "$(CUR_DIR)/magpie/magpie.ini" --workers 10 --preload &'
```

Comment:
That little & at the end of the command looks like a hack, no?

Reply:
Tells bash to not wait for the return of the call.
So basically, run the process detached.